applications/OptimizationApplication/custom_external_libraries/nlopt/COPYING

```diff
@@ -0,0 +1,40 @@
+NLopt combines several free/open-source nonlinear optimization
+libraries by various authors.  See the COPYING, COPYRIGHT, and README
+files in the subdirectories for the original copyright and licensing
+information of these packages.
+
+The compiled NLopt library, i.e. the combined work of all of the
+included optimization routines, is licensed under the conjunction of
+all of these licensing terms.  Currently, the most restrictive terms
+are for the code in the "luksan" directory, which is licensed under
+the GNU Lesser General Public License (GNU LGPL), version 2.1 or
```

Comment:
This is not compatible with the license of Kratos

Reply:
It is the same as MMG, please follow same approach

Reply:
yes. Thanks for the hint. I will check and follow MMG work.